src/span.js

```diff
@@ -36,6 +36,7 @@ export default class Span {
     _tags: Array<Tag>;
     static _baggageHeaderCache: any;
```

Comment:
+1, I did't even know JS had a Map. But I'd leave it out of this diff, it sounds like a global refactoring